src/TorchSharp/NN/Utils/PackedSequence.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) .NET Foundation and Contributors.  All Rights Reserved.  See LICENSE in the project root for license information.
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace TorchSharp
+{
+    public static partial class torch
+    {
+        public static partial class nn
+        {
+            public static partial class utils
+            {
+                public static partial class rnn
+                {
+                    /// <summary>
+                    /// A packed batch of variable length sequences.
+                    /// </summary>
+                    public class PackedSequence : IDisposable
```

Comment:
>PackedSequence [](http://example.com/codeflow?start=33&length=14)

can we seal this class? or it expected to have someone subclass it in some scenarios?

Reply:
Thanks. Added seal.